cogboard-webapp/src/components/FormMessages.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from '@emotion/styled/macro';
+
+
+const FormMessages = ({className, messages, ...others}) => {
```

Comment:
I think naming it ValidationMessages will be better.

Reply:
done